app/components/form_components/autocomplete/script.js

```diff
@@ -0,0 +1,17 @@
+import './style.scss'
+import accessibleAutocomplete from 'accessible-autocomplete'
+import { nodeListForEach } from 'govuk-frontend/govuk/common'
+
+const $allAutocompleteElements = document.querySelectorAll('[data-module="app-autocomplete"]')
+const showAllValuesOption = component => (component.getAttribute('data-show-all-values') && 'true') === 'true'
```

Comment:
do we need to convert with `Boolean()` to treat these as proper boolean values?

Reply:
🤦